### DISCUSSION TITLE
Elden Ring V.1.05 Cheat Not Working

### DISCUSSION THREAD

**Discussion by kocakli** [GoldHEN/GoldHEN_Cheat_Repository#38] [Q&A] [answered]
Hello Elden Ring CUSA28863 v01.05 Cheat didn't work on my 9.0 PS4.
Uploaded JSON with FTP but never worked. Do u have any solutions?

**Comment by kmeps4** [comment, marked as answer]
https://github.com/GoldHEN/GoldHEN_Cheat_Repository/discussions/42#discussion-4222094
